src/style.css

```diff
@@ -116,40 +121,39 @@ main .grid-item {
 }
 
 .grid-container input[type="button"] {
-  width: 100%;
-  border: 1px black solid;
-  line-height: 1em;
-  outline: none;
-  font-size: large;
-  color: gray;
-  padding: 0.5em;
+  width: 100% !important;
+  border: 1px black solid !important;
+  line-height: 1em !important;
+  outline: none !important;
+  font-size: large !important;
+  color: gray !important;
+  padding: 0.5em !important;
 }
 
 .grid-container input[type="button"]:hover {
-  outline: none;
-  background: rgb(205, 158, 41);
-  color: white;
+  outline: none !important;
+  background: rgb(205, 158, 41) !important;
+  color: white !important;
```

Comment:
## Optional Suggestion

- To make your code base DRY and easy to debug, avoid so much use of !important since this overrides every line of codes it's added to.